src/utils/common.ts

```diff
@@ -195,9 +196,14 @@ export async function fetchJson<T = any>(
  * Prepare human-readable name with protocol for an account.
  * E.g.: `Æternity account 1`, `Bitcoin account 2`
  */
-export function getDefaultAccountLabel({ protocol, idx }: Partial<IAccount> = {}): string {
+
+export function getDefaultAccountLabel(
+  { protocol, idx, type }: Partial<IAccount> = {},
+): string {
+  const isAirGap = type === ACCOUNT_TYPES.airGap;
   return [
-    (protocol) ? ProtocolAdapterFactory.getAdapter(protocol).protocolName : null,
+    (protocol && !isAirGap) ? ProtocolAdapterFactory.getAdapter(protocol).protocolName : null,
+    (isAirGap) ? tg('airGap.title') : null,
```

Comment:
As I like to be future proof and I don't like to repeat the same statements I suggest to add a switch..case statement to distinguish the account types:

```
  const accountTypeName = (() => {
    switch (true) {
      case type === ACCOUNT_TYPES.airGap: return tg('airGap.title');
      default: return protocol ? ProtocolAdapterFactory.getAdapter(protocol).protocolName : null;
    }
  })();
  return [
    accountTypeName,
    tg('pages.account.heading'),
    (idx || 0) + 1,
  ]
    .filter(excludeFalsy)
    .join(' ');
```

Not 100% sure about this, but you see where I'm going? What do you think?